modules/customer.inc.php

```diff
@@ -124,6 +124,13 @@
         'short' => true,
     );
 
+    $alltickets = (isset($_GET['alltickets']) && !empty($_GET['alltickets']))
+        || ((!isset($_GET['alltickets']) && ConfigHelper::checkConfig('phpui.default_show_closed_tickets')));
+    if ($alltickets) {
```

Comment:
Skoro zawsze wymagasz, żeby `$_GET['alltickets']` było ustawione to po co za każdym razem sprawdzasz czy jest ustawione.

Reply:
Nie zastanawiałem się nad tym głębiej skopiowałem z innej zakładki kod. 
Czy takie uproszczenie wystarczy?

Reply:
Przebudowałem ten test: 355372e7ebdfe34569651e1cd24a0141d6ac08e0
Tak jest wystarczająco?